### ISSUE TITLE
Wrong generation of generics

### ISSUE THREAD

**Issue by fpotera** [stoicflame/enunciate#580]
Hello.
We generate api for below class:

```java
import java.util.List;

 public class ActionResult<T>
 {
	private T bean;

	private List<ResponseMessage> messages;

	public static <T> ActionResult<T> create(T bean, List<ResponseMessage> messages)
	{
		return new ActionResult<T>(bean, messages);
	}

	public ActionResult()
	{
	}

	private ActionResult(T bean)
	{
		this.bean = bean;
	}

	private ActionResult(T bean, List<ResponseMessage> messages)
	{
		this.bean = bean;
		this.messages = messages;
	}

	public T getBean()
	{
		return bean;
	}

	public List<ResponseMessage> getMessages()
	{
		return messages;
	}
 }
```

In swagger.json file is generated as:

```json
    "json_ActionResult" : {
      "type" : "object",
      "title" : "ActionResult",
          "properties" : {
            "bean" : {
"description" : "",
"type" : "file"
            },
            "messages" : {
"description" : "",
"type" : "array",
"items" : {
"type" : "array",
"items" : {
  "$ref" : "#/definitions/json_ResponseMessage"
}
}
            }
          },
      "description" : ""
    }
```

For "bean" property "file" type is wrong. I think "object" is better.

Am I right?

And also for collection "array" in "array" of ResponseMessage, should be just an simple "array".

**Comment by stoicflame** [comment]
Fixed in 2.9

**Comment by stoicflame** [comment]
[Enunciate 2.9.0 has been released.](https://github.com/stoicflame/enunciate/releases/tag/v2.9.0)
